newIDE/app/src/BrowserApp.js

```diff
@@ -50,6 +50,7 @@ export const create = (authentification: Authentification) => {
         filterExamples: !Window.isDev(),
       })}
       initialPathsOrURLsToOpen={appArguments['_']}
+      cmdArguments={appArguments}
```

Comment:
So this works! But I think we're breaking a bit the encapsulation by giving too much information to MainFrame. This poor MainFrame should not have to deal with an Object containing ALL the arguments. This should be the responsibility of BrowserApp and LocalApp to read the arguments, and pass it into well named props of MainFrame that have a behavior that is obvious just by reading the name of the prop.

Why is this important? Because ideally, you pass the **minimum of information** down to your components. This ease refactoring. Let's someone want to refactor the arguments in the future.
* If you pass only a few props based on the arguments on BrowserApp and LocalApp, **only 2 files** are to be refactored.
* If you pass the whole "cmdArguments", then potentially it is possible that MainFrame is using it for tons of things, doing black magic with it, making things according to arguments but not other, etc, etc, etc... Lots of uncertainty!

What I recommend is to simply pass:
`disableCheckForUpdates={!!appArguments['disable-update-check']}`.

This means that:
* it's easy to know what the arguments are doing by just reading `LocalApp.js` or `BrowserApp.js` - the source code is self documenting in one place!
* if at some point we want to disable check for updates for another reason, we just have to add another condition here!

Reply:
thank you for taking the time to explain :) I applied the requested changes now